integration-tests/tests/login_test.go

```diff
@@ -139,38 +175,23 @@ func ipTablesCommand(action, serverAddrPort string) []string {
 		"--to-destination", serverAddrPort}
 }
 
-func (s *loginSuite) writeCredentials(loginName string) error {
-	cmds, _ := cli.AddOptionsToCommand([]string{"sudo", "snap", "login", loginName})
-	cmd := exec.Command(cmds[0], cmds[1:]...)
-	f, err := pty.Start(cmd)
-	if err != nil {
-		return err
-	}
-
-	err = cmdInteract(f, "Password: ", password)
-	if err != nil {
-		return err
-	}
-	io.Copy(&s.stdout, f)
+var _ = check.Suite(&storeSuite{})
 
-	return nil
+type storeSuite struct {
+	common.SnappySuite
 }
 
-func cmdInteract(f *os.File, prompt, input string) error {
-	buf := make([]byte, len(prompt))
+func (s *storeSuite) TestStoreLoginLogout(c *check.C) {
+	username := os.Getenv("TEST_USER_NAME")
+	password := os.Getenv("TEST_USER_PASSWORD")
 
-	_, err := io.ReadFull(f, buf)
-	if err != nil {
-		return err
+	if username == "" || password == "" {
+		c.Skip("Test user credentials not available in current environment, skipping store tests")
```

Comment:
I suggest this instead:

`c.Skip("$TEST_USER_NAME and $TEST_USER_PASSWORD not defined")

The message shows up next to the test actually skipped already, so no need to call it out.